sap_business_document_processing/document_classification_client/constants.py

```diff
@@ -22,13 +22,21 @@
 API_PAGINATION_SKIP_PARAM = 'skip'
 API_PAGINATION_COUNT_PARAM = 'count'
 
-API_MIME_TYPE_FIELD = 'mimeType'
+API_DATASET_ID_FIELD = 'datasetId'
+API_DATASETS_FIELD = 'datasets'
+API_DEPLOYMENT_ID_FIELD = 'deploymentId'
+API_DEPLOYMENTS_FIELD = 'deployments'
 API_DOCUMENT_EXTRACTED_TEXT_FIELD = 'extractedText'
 API_DOCUMENT_ID_FIELD = 'documentId'
-API_STATUS_FIELD = 'status'
+API_DOCUMENT_FIELD = 'document'
+API_GROUND_TRUTH_FIELD = 'groundTruth'
+API_MIME_TYPE_FIELD = 'mimeType'
+API_MODELS_FIELD = 'models'
+API_MODEL_NAME_FIELD = 'modelName'
+API_MODEL_VERSION_FIELD = 'modelVersion'
+API_PARAMETERS_FIELD = 'parameters'
+API_RESULTS_FIELD = 'results'
+API_STRATIFICATION_SET_FIELD = 'stratificationSet'
 
 # NOTE: Equivalent uppercase/mixed case will be picked up as well
 FILE_EXTENSIONS_FOR_FOLDER_UPLOAD = ('.pdf', '.png', '.jpeg', '.jpg')
```

Comment:
Can the stratification_set parameter then also be removed from [this ](https://github.com/SAP/business-document-processing/blob/6e41abbee7ac6206718f81cf2f600ccd5c100202/sap_document_classification_client/dc_api_client.py#L258) function?

Reply:
No, we use the `stratification_set` at least in tests.

Reply:
I think I misread the comment. Do you mean that the FILE_EXTENSIONS_FOR_FOLDER_UPLOAD constant is not used anymore? Because it is part of the _find_files function

Reply:
I see, I still think we need to remove it as we are removing file type validation from DOX as well, we just need to mention in the function description that the user has to take care about the files format to be supported.

Reply:
Constant has been removed. Default for discovered data files is now '.pdf'